src/state.ts

```diff
@@ -11,7 +10,11 @@ export interface State {
   prismaticUrl: string;
   screenConfiguration?: ScreenConfiguration;
   theme?: Theme;
-  fontConfiguration?: Pick<WebFont.Config, "google">;
+  fontConfiguration?: {
+    google: {
+      families: Array<string>;
+    };
+  };
```

Comment:
Lets make this into an interface and export it so the users can use it if needed when consuming the embedded project. Exactly like Screen Configuration, Filters, Translation, Theme and so on.

Reply:
@jasoncomes good call. Does the latest commit look more correct?